composer.json

```diff
@@ -28,7 +28,10 @@
 	},
 	"require": {
 		"php": ">=7.4",
-		"composer/installers": ">=1.0.1"
+		"composer/installers": ">=1.0.1",
+		"mediawiki/semantic-media-wiki": "~3.1|~4.0|~5.0",
```

Comment:
Remove this please. It’s unneeded and could be why ci is failing.

Reply:
done